### DISCUSSION TITLE
Sync issue everyday on the afternoon for about 4 hours

### DISCUSSION THREAD

**Discussion by penafree** [Chia-Network/chia-blockchain#8405] [Support]
Hello all!

I have strange sync issues (red not Synced message) everyday on the afternoon for about 4 hours after that everything is normal and it syncs again. This never happened to me before, is it because the net space is largely increasing? I have a good connection and port 8444 is opened on my router. 
Could someone help me on this?

Best regards,
António Pena

**Comment by BrandtH22** [comment]
@penafree , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
